widget/charts/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@rango-dev/charts",
+  "version": "0.1.0",
+  "license": "MIT",
+  "type": "module",
+  "source": "./src/index.ts",
+  "main": "./dist/index.js",
+  "exports": {
+    "types": "./dist/index.d.ts",
+    "default": "./dist/index.js"
+  },
+  "typings": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "dev": "yarn bundle --watch",
+    "build": "node ../../scripts/build/command.mjs --path widget/charts",
+    "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
+    "type-checking": "tsc --declaration --emitDeclarationOnly",
+    "clean": "rimraf dist",
+    "format": "prettier --write '{.,src}/**/*.{ts,tsx}'",
+    "lint": "eslint \"**/*.{ts,tsx}\" --ignore-path ../../.eslintignore"
+  },
+  "peerDependencies": {
+    "@lingui/core": "4.2.1",
+    "@lingui/react": "4.2.1",
+    "react": ">=16"
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.8",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "dependencies": {
+    "rango-types": "^0.1.69",
+    "@visx/curve": "^2.1.0",
+    "@visx/gradient": "^3.3.0",
+    "@visx/xychart": "^2.16.0",
```

Comment:
We did not use `@visx/xychart` in the `BarChart`,
Please add only the items we need.

Reply:
fixed